packages/shopping/src/application.ts

```diff
@@ -47,6 +48,13 @@ export class ShoppingApplication extends BootMixin(
 ) {
   constructor(options?: ApplicationConfig) {
     super(options);
+    this.api({
+      openapi: '3.0.0',
+      info: {title: '', version: ''},
```

Comment:
Can you fill `title` and `version` with something more meaningful please? Personally, I'd use metadata from `package.json`.

```ts
import * as package from '../package.json';
// ...

info: {title: package.name, version: package.version}
```

Reply:
updated 👌